lending-pool/src/auctions/bad_debt_auction.rs

```diff
@@ -748,7 +748,7 @@ mod tests {
             &vec![&e, 500_001_0000000, 12_501_0000000],
             &samwise,
         );
-        backstop_client.deposit(&samwise, &pool_address, &150_0000000);
+        backstop_client.deposit(&samwise, &pool_address, &2_000_0000000);
```

Comment:
Can we add a `test_fill_bad_debt_auction_leftover_debt_backstop_past_threshold` that has a, say, 5% backstop threshold and does not burn?

Reply:
see `test_fill_bad_debt_auction_leftover_debt_sufficient_balance`